src/Helpers.hpp

```diff
@@ -0,0 +1,75 @@
+#ifndef __RADAR_BASE_HELPERS_HPP__
+#define __RADAR_BASE_HELPERS_HPP__
+
+#include "EchoToImageLUT.hpp"
+#include "Radar.hpp"
+#include <base/Angle.hpp>
+#include <base/Float.hpp>
+#include <base/Time.hpp>
+
+#include <vector>
+namespace radar_base {
+    namespace helpers {
+
+        /**
+         * Function to be iterated inside each sweep, its inputs are the radar echo
+         * itself, the current local echo sweep index as well as the global sweep angular
+         * position.
+         */
+        using sweepFunction = std::function<void(Radar const&, int, int)>;
```

Comment:
Give name to parameters so it is easy to one understand what data is passed to the function. Also this define "kind of" type, shouldn't it be **PascalCase** ?
```suggestion
        using SweepFunction = std::function<void(Radar const& echo, int sweep_local_index, int sweep_global_index)>;
```